xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -512,6 +516,70 @@ rebuild_master:
         configure_repositories step, set the <emphasis>add_repositories</emphasis> key to anything else than <emphasis>enabled</emphasis>.</para>
         </sect4>
 
+        <sect4 xml:id="sec.obs.obs_scm_ci_workflow_integration.obs_workflows.steps.submit_request">
+          <title>Submit a Request</title>
+
+          <para>The submit request step is the equivalent of the <emphasis>osc submitrequest</emphasis> command.</para>
+
+          <para>The requirements to run a submit request step are:</para>
+
+          <itemizedlist>
+            <listitem>
+              <para>There has to be a package branched already.</para>
+            </listitem>
+
+            <listitem>
+              <para>There has to be some changes in that branched package.</para>
+            </listitem>
+          </itemizedlist>
+
+          <para>After the previous requirements are met, keep in mind that:</para>
+
+          <itemizedlist>
+            <listitem>
+              <para>With a pull request open event, push event, or tag_push event, it will:</para>
+              <itemizedlist>
+                <listitem>
+                  <para>create the submit request.</para>
+                </listitem>
+
+                <listitem>
+                  <para>report further request status changes.</para>
```

Comment:
There is not reporting for `tag_push`. I think you should leave the reporting for the reference table.

Reply:
Alright. I removed this sentence here, and also from the `tag_push` event in the reference table. Please take a look again.